### ISSUE TITLE
Issue compiling basic_dsp (SIMD)

### ISSUE THREAD

**Issue by Camotubi** [liebharc/basic_dsp#40]
Hi,
I wasnt able to compile basic_dsp. 
Since I wasn't able to find these fix that @liebharc commented on [rust-lang issue tracker](https://github.com/rust-lang/rust/issues/57940) I'll place it here. If you are having this errors when compiling  add these to your cargo toml.
```
[dependencies.basic_dsp]
version="0.7.1"
default-features = false
features = ["std"]
```

Also,
I saw this [commit](https://github.com/liebharc/basic_dsp/commit/300a93798874078fb5024e61c09586826ef53089) that removes simd from the default features of basic_dsp vector . The issue is that basic dsp ( parent crate) still has it by default, which leads to the compilation to fail.

**Comment by liebharc** [comment]
Hi, 
Thanks for pointing that out and providing a solution :). I am travelling right now so it will take a while before I am able to fix that. If you like, feel free to open a PR for the parent Cargo.toml.

Am 15. Februar 2019 08:54:29 GMT+13:00 schrieb "Carlos Tuñón" <notifications@github.com>:

**Comment by Camotubi** [comment]
Done :)

**Comment by liebharc** [comment]
Should be closed with [0.7.2](https://crates.io/crates/basic_dsp/0.7.2). Also thanks for your usability comment on the other issue. I think feedback like that is what this lib needs most at the moment.
